src/traces/parcats/parcats.js

```diff
@@ -238,11 +238,7 @@ function performPlot(parcatsModels, graphDiv, layout, svg) {
             })
         .attr('alignment-baseline', 'middle')
 
-        .style('text-shadow',
-            paperColor + ' -1px  1px 2px, ' +
-            paperColor + ' 1px  1px 2px, ' +
-            paperColor + '  1px -1px 2px, ' +
-            paperColor + ' -1px -1px 2px')
+        .style('text-shadow', svgTextUtils.makeTextShadow(1, 1, 2, paperColor))
```

Comment:
Interesting to see that `parcats` uses different "2" for `blur-radius` instead of 1.
@alexcjohnson do you think we should declare and unify this in a new layout container?

Reply:
Good 👁️ 
To me the 1 vs 2 doesn't make much visible difference, I'd probably standardize on 1. Also given that the purpose here is really outlining rather than a 3D-like shadow I'd combine `offsetX` and `offsetY` into a single `offset`.

Reply:
I'm fine switching to 1 and regenerating the parcats baselines. And fine with combining the offsets.